package.json

```diff
@@ -81,8 +81,8 @@
   },
   "dependencies": {
     "@bem-react/classname": "^1.6.0",
-    "@gravity-ui/components": "^2.4.0",
-    "@gravity-ui/dynamic-forms": "^2.3.0",
+    "@gravity-ui/components": "^3.0.0",
+    "@gravity-ui/dynamic-forms": "^4.0.0",
     "@gravity-ui/i18n": "^1.0.0",
```

Comment:
I suggest you to install the latest version of i18n because of usage it in uikit https://github.com/gravity-ui/uikit/blob/main/package.json#L95